res_users_password_security/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+==================
+Password  Security
```

Comment:
I think module technical name could be 'password_security'. 
Display name and technical name should be the same (except plural maybe), I think at least.
2 words is quickly read, and when we read password, everybody think about users.